### PR TITLE
Update vn_fptplay.m3u

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by LaneSh4d0w** [comment]
Neither of those are online.

**Review by freearhey — COMMENTED** [review]
image

**Comment by bbbureneecartoonz** [comment]
> image

waht is this software??

**Comment by freearhey** [comment]
@bbbureneecartoonz 
>waht is this software??

https://streamtest.in/

File: streams/vn_fptplay.m3u [hunk 1]
```diff
@@ -56,9 +59,15 @@ https://livecdn.fptplay.net/hda1/blueantent_vhls.smil/chunklist.m3u8
...
+#EXTVLCOPT:http-user-agent=Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/92.0.4515.131 Safari/537.36
+https://livecdn.fptplay.net/sdb/vtc1_hls.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="VTC2.vn" status="online" user-agent="Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/92.0.4515.131 Safari/537.36",VTC2 (720p) [Geo-blocked]
+#EXTVLCOPT:http-user-agent=Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/92.0.4515.131 Safari/537.36
+https://livecdn.fptplay.net/sdb/vtc2_hls.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="VTC3.vn" status="online" user-agent="Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/92.0.4515.131 Safari/537.36",VTC3 (720p) [Geo-blocked]
 #EXTVLCOPT:http-user-agent=Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/92.0.4515.131 Safari/537.36
-https://vips-livecdn.fptplay.net/hda1/vtc1_vhls.smil/playlist.m3u8
+https://livecdn.fptplay.net/sdb/vtc3_hls.smil/playlist.m3u8
```
Comment (freearhey) [inline streams/vn_fptplay.m3u:70]: 404 Not Found

File: streams/vn_fptplay.m3u [hunk 2]
```diff
@@ -68,6 +77,15 @@ https://livecdn.fptplay.net/sdb/vtc5_hls.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="VTC6.vn" status="online" user-agent="Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/92.0.4515.131 Safari/537.36",VTC6 (576p)
 #EXTVLCOPT:http-user-agent=Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/92.0.4515.131 Safari/537.36
 https://livecdn.fptplay.net/sdb/vtc6_hls.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="VTC7.vn" status="online" user-agent="Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/92.0.4515.131 Safari/537.36",VTC7 (576p)
+#EXTVLCOPT:http-user-agent=Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/92.0.4515.131 Safari/537.36
+https://livecdn.fptplay.net/sdb/vtc7_hls.smil/playlist.m3u8
```
Comment (freearhey) [inline streams/vn_fptplay.m3u:82]: 404 Not Found

File: streams/vn_fptplay.m3u [hunk 3]
```diff
@@ -68,6 +77,15 @@ https://livecdn.fptplay.net/sdb/vtc5_hls.smil/playlist.m3u8
...
+#EXTINF:-1 tvg-id="VTC8.vn" status="online" user-agent="Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/92.0.4515.131 Safari/537.36",VTC8 (576p)
+#EXTVLCOPT:http-user-agent=Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/92.0.4515.131 Safari/537.36
+https://livecdn.fptplay.net/sdb/vtc8_hls.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="VTC9.vn" status="online" user-agent="Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/92.0.4515.131 Safari/537.36",VTC9 (576p)
+#EXTVLCOPT:http-user-agent=Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/92.0.4515.131 Safari/537.36
+https://livecdn.fptplay.net/sdb/vtc9_hls.smil/playlist.m3u8
```
Comment (freearhey) [inline streams/vn_fptplay.m3u:88]:
```
<html>
<head><title>403 Forbidden</title></head>
<body>
<center><h1>403 Forbidden</h1></center>
<hr><center>nginx</center>
</body>
</html>
```

File: streams/vn_fptplay.m3u [hunk 4]
```diff
@@ -38,6 +38,9 @@ https://vips-livecdn.fptplay.net/hda1/hbo_vhls.smil/chunklist.m3u8
 #EXTINF:-1 tvg-id="HTV1.vn" status="online" user-agent="Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/92.0.4515.131 Safari/537.36",HTV1 (576p)
 #EXTVLCOPT:http-user-agent=Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/92.0.4515.131 Safari/537.36
 https://livecdn.fptplay.net/sdb/htv1_hls.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="HTV2.vn" status="online" user-agent="Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/92.0.4515.131 Safari/537.36",HTV2 (576p)
+#EXTVLCOPT:http-user-agent=Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/92.0.4515.131 Safari/537.36
+https://livecdn.fptplay.net/sdb/htv2_hls.smil/playlist.m3u8
```
Comment (freearhey) [inline streams/vn_fptplay.m3u:43]: 404 Not Found

**Review by freearhey — CHANGES_REQUESTED** [review]
Please check the marked links again.

**Review by ghost — COMMENTED** [review]
bb

**Comment by LaneSh4d0w** [comment]
Closed due to lack of activity.

**Comment by bbbureneecartoonz** [comment]
reopen pls

**Review by ghost — COMMENTED** [review]
bb

**Comment by bbbureneecartoonz** [comment]
merge pls

**Comment by freearhey** [comment]
we cannot approve this PR until you remove all broken links from it

File: streams/vn_fptplay.m3u (hunk 1 above)
Comment (LaneSh4d0w) [reply streams/vn_fptplay.m3u:64]: now 403s, certainly geo-blocked.

File: streams/vn_fptplay.m3u [hunk 5]
```diff
@@ -68,6 +77,15 @@ https://livecdn.fptplay.net/sdb/vtc5_hls.smil/playlist.m3u8
...
 #EXTVLCOPT:http-user-agent=Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/92.0.4515.131 Safari/537.36
 https://livecdn.fptplay.net/sdb/vtc6_hls.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="VTC7.vn" status="online" user-agent="Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/92.0.4515.131 Safari/537.36",VTC7 (576p)
+#EXTVLCOPT:http-user-agent=Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/92.0.4515.131 Safari/537.36
+https://livecdn.fptplay.net/sdb/vtc7_hls.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="VTC8.vn" status="online" user-agent="Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/92.0.4515.131 Safari/537.36",VTC8 (576p)
+#EXTVLCOPT:http-user-agent=Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/92.0.4515.131 Safari/537.36
+https://livecdn.fptplay.net/sdb/vtc8_hls.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="VTC9.vn" status="online" user-agent="Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/92.0.4515.131 Safari/537.36",VTC9 (576p)
```
Comment (LaneSh4d0w) [inline streams/vn_fptplay.m3u:80]:
```suggestion
#EXTINF:-1 tvg-id="VTC9.vn" status="online" user-agent="Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/92.0.4515.131 Safari/537.36",VTC9 (576p) [Geo-blocked]
```

**Review by BellezaEmporium — APPROVED** [review]

**Comment by LaneSh4d0w** [comment]
Marked the 403 links as certainly geo-blocked. I don't have any Vietnamese VPN, but we sure have a big community that can tell us if links work or not.

**Review by freearhey — APPROVED** [review]

**Pull request merged**